app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java

```diff
@@ -0,0 +1,241 @@
+        super.onCreate(savedInstanceState);
+
+        if (savedInstanceState != null && savedInstanceState.getString(SEARCH) != null) {
+            searchText = savedInstanceState.getString(SEARCH);
+        }
+        if (getArguments() != null) {
+            showBookmarkedOnly = getArguments().getBoolean(SHOW_BOOKMARKED_ONLY, false);
+        }
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View v = inflater.inflate(R.layout.fragment_schedule, container, false);
+        scheduleTitlePicker = (TitlePickerView) v.findViewById(R.id.schedule_title_picker);
+        scheduleViewPager = (ViewPager) v.findViewById(R.id.schedule_view_pager);
+        viewPagerAdapter = new ScheduleViewPagerAdapter(getChildFragmentManager());
+        scheduleViewPager.setAdapter(viewPagerAdapter);
+        scheduleTitlePicker.setupWithViewPager(scheduleViewPager);
+        return v;
+    }
+
+    @Override
+    public void onSaveInstanceState(Bundle bundle) {
+        if (isAdded()) {
+            if (searchView != null) {
+                bundle.putString(SEARCH, searchText);
+            }
+        }
+        super.onSaveInstanceState(bundle);
+    }
+
+    private class ScheduleViewPagerAdapter extends FragmentPagerAdapter {
+        private String[] titles;
+        private SparseArray<Fragment> registeredFragments = new SparseArray<>();
+
+        public ScheduleViewPagerAdapter(FragmentManager fm) {
+            super(fm);
+            titles = getResources().getStringArray(R.array.days);
+        }
+
+        @Override
+        public Fragment getItem(int position) {
+            Fragment fragment = new SessionFragment();
+            Bundle bundle = new Bundle();
+            bundle.putInt(SCHEDULE_TAB_POSITION, position);
+            bundle.putBoolean(SHOW_BOOKMARKED_ONLY, showBookmarkedOnly);
+            fragment.setArguments(bundle);
+            return fragment;
+        }
+
+        @Override
+        public Object instantiateItem(ViewGroup container, int position) {
+            Fragment fragment = (Fragment) super.instantiateItem(container, position);
+            registeredFragments.put(position, fragment);
+            return fragment;
+        }
+
+        @Override
+        public void destroyItem(ViewGroup container, int position, Object object) {
+            registeredFragments.remove(position);
+            super.destroyItem(container, position, object);
+        }
+
+        public Fragment getRegisteredFragment(int position) {
+            return registeredFragments.get(position);
+        }
+
+        @Override
+        public int getCount() {
+            return titles.length;
+        }
+
+        @Override
+        public CharSequence getPageTitle(int position) {
+            return titles[position];
+        }
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://www.codacy.com/app/triplez/open-event-android/file/2406802117/issues/source?bid=3233850&fileBranchId=3233911#l135)